app/views/bulkrax/importers/_form.html.erb

```diff
@@ -23,6 +23,11 @@
 
   <%= form.input :parser_klass, collection: Bulkrax.parsers.map {|p| [p[:name], p[:class_name], {'data-partial' => p[:partial]}]}, label: "Parser", input_html: { class: 'form-control' } %>
 
+  <%= form.input :import_relationships_only?,
```

Comment:
I think you likely won't want the trailing `?` ; I don't know if it will attempt to cast that method to `:import_relationships?=`